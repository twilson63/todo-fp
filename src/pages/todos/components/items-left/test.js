/* global test, expect */
import ItemsLeft from './'
import React from 'react'
import { shallow } from 'enzyme'

test('itemsLeft Component renders correctly', done => {
  const result = shallow(
    <ItemsLeft todos={[]} />
  )

  expect(result).toMatchSnapshot()
  done()
})
