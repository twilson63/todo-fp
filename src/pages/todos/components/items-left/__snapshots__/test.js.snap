exports[`test itemsLeft Component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <span
    className="todo-count">
    <strong />
     item left
</span>,
  "nodes": Array [
    <span
      className="todo-count">
      <strong />
       item left
</span>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <ItemsLeft
        todos={Array []} />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "todos": Array [],
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <span
          className="todo-count">
          <strong />
           item left
</span>,
        "_debugID": 2,
        "_renderedOutput": <span
          className="todo-count">
          <strong />
           item left
</span>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ItemsLeft
    todos={Array []} />,
}
`;
